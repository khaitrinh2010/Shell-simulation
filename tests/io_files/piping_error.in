echo "Khai" | cat ||
sort students.csv |   | cat
