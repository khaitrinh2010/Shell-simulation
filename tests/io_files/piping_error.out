>> mysh: syntax error: expected command after pipe
>> mysh: syntax error: expected command after pipe
>> 
